learning_journal_basic/setup.py

```diff
@@ -10,17 +10,31 @@
 
 requires = [
     'pyramid',
-    'pyramid_chameleon',
     'pyramid_debugtoolbar',
     'waitress',
+    'pyramid_jinja2',
+    'pyramid_tm',
+    'SQLAlchemy',
+    'transaction',
+    'zope.sqlalchemy',
+    'faker',
+    'ipython',
+    'pyramid-ipython',
```

Comment:
You have ipython and pyramid-ipython here as well as in the development_extra.  You only need one set (development_extra is probably a better place since the app doesn't require these)